specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -216,6 +269,24 @@
           }
         }
       }
+    },
+    "RoutingForNumber": {
+      "description": "Represents number routing validation details.",
+      "type": "object",
+      "properties": {
+        "targetNumber": {
+          "description": "A string in the E.164 format of a phone number, e.g. +11234567890.",
+          "type": "string"
+        },
+        "matchingRoutes": {
+          "description": "The list of routes whose number patterns are matched by the target number. The routes are displayed and apply in the same order as in SipConfiguration. Only route names are displayed.",
```

Comment:
The discription should not talk about how numbers are displayed in the UI. In particular "Only route names are displayed" is specific to how the API is called and responses used in the UI.

Reply:
Removed the sentence.